client/homebrew/utils/versionHistory.js

```diff
@@ -0,0 +1,123 @@
+export const HISTORY_PREFIX = 'HOMEBREWERY-HISTORY';
+export const HISTORY_SLOTS = 5;
+
+const DEFAULT_HISTORY_SAVE_DELAYS = {
+ 	'0' : 0,           // 0 minutes (if not specified)
+	'1' : 2,			// 2 minutes
+	'2' : 10,			// 10 minutes
+	'3' : 60,			// 60 minutes
+	'4' : 12 * 60,     // 12 hours
+	'5' : 2 * 24 * 60  // 2 days
+};
```

Comment:
Use consistent tabs/ spaces between rows

Reply:
I did, then commented it, then uncommented it... I don't think the Linter catches spacing before comments.

Reply:
Actually, I'll just take those comments out and add one that specifies that the values are in minutes. I think `2 * 24 * 60` makes it clear that the value is 2 days; it would might require clarification if the value was just `2880`.